### ISSUE TITLE
I'm having the issue of missing /VERSION file in a vanilla install of Python

### ISSUE THREAD

**Issue by captainproton795** [AtsushiSakai/PyJSONViewer#12]
I have a fresh install of Python 3.8 x64 on Windows 10

I have run pip install pyjsonviewer and it looked good.

I go to run pyjsonviewer and receive error:

**PS C:\PYTHON38-64\Lib\site-packages\pyjsonviewer> py .\pyjsonviewer.py**
_Traceback (most recent calllast):
   File ".\pyjsonviewer.py", line 20, in <module>
       VERSION = open(os.path.dirname(os.path.abspath(__file__))+"/VERSION", "r").readline()
FileNotFoundError:  [Errno 2] No such file or directory:  'C:\\PYTHON38-64\\Lib\\site-packages\\pyjsonviewer/VERSION'_

**Comment by ferdymercury** [comment]
Me too in Linux Mint Mate after a fresh install from pip3:
```
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/usr/local/lib/python3.6/dist-packages/pyjsonviewer/__main__.py", line 1, in <module>
    from pyjsonviewer.pyjsonviewer import main
  File "/usr/local/lib/python3.6/dist-packages/pyjsonviewer/pyjsonviewer.py", line 20, in <module>
    VERSION = open(os.path.dirname(os.path.abspath(__file__))+"/VERSION", "r").readline()
FileNotFoundError: [Errno 2] No such file or directory: '/usr/local/lib/python3.6/dist-packages/pyjsonviewer/VERSION'

```

I solved it by creating a file called VERSION in that directory with a dummy number inside.

**Comment by AtsushiSakai** [comment]
@divinehand @ferdymercury Thank you for report this, and I'm so sorry for that. I released v1.4.4.
https://pypi.org/project/PyJSONViewer/
I believe this problem is fixed in the newest version. Please let me know when you find this issue again.
